### PR TITLE
Support deinit_existential_addr

### PR DESCRIPTION
**Pull request by bryantam**
Issue #98

### CODE REVIEW

File: lib/WALASupport/SILWalaInstructionVisitor.cpp
Comment (karimhamdanali) [inline lib/WALASupport/SILWalaInstructionVisitor.cpp]: Please update the return value to not use `nullptr`.

**Review by karimhamdanali — CHANGES_REQUESTED** [review]
Please update the return value to not use `nullptr`.

**Comment by bryantam** [comment]
Empty node is now returned. But I think we should wait until #106 merged, as the new organization might be messed up

**Comment by karimhamdanali** [comment]
@bryantam, could you please resolve the conflicts now after merging Mark's PR?

**Comment by bryantam** [comment]
It's ready now

**Comment by karimhamdanali** [comment]
Hmm, it still shows some conflicts :S That’s a bit weird, given that the other PR didn’t have any merge conflict issues when I just approved it moments ago. Could you please resolve this one as well?

Thanks!

**Comment by bryantam** [comment]
oops... It's actually conflicting with my own PR. Fixed.

**Review by karimhamdanali — APPROVED** [review]

**Pull request merged**
